holmes/plugins/toolsets/prometheus.py

```diff
@@ -74,18 +76,17 @@ def result_has_data(result: dict) -> bool:
     return False
 
 
-def fetch_metrics_labels(
+def fetch_metrics_labels_with_series_api(
     prometheus_url: str,
     cache: Optional[TTLCache],
     metrics_labels_time_window_hrs: Union[int, None],
     metric_name: str,
 ) -> dict:
     """This is a slow query. Takes 5+ seconds to run"""
-    cache_key = f"metrics_labels:{metric_name}"
+    cache_key = f"metrics_labels_series_api:{metric_name}"
     if cache:
         cached_result = cache.get(cache_key)
         if cached_result:
-            logging.info("fetch_metrics_labels() result retrieved from cache")
```

Comment:
Removed the logging to stay consistent across both functions `fetch_metrics_labels_with_series_api` and `fetch_metrics_labels_with_labels_api`.